website/docs-programming/04 - Parameters.md

```diff
@@ -127,7 +127,7 @@ cut.redundancies | bool  |  integrate redundancy checking of cuts | true
 cut.xor | bool  |  extract xors from clauses for cut simplification | false
 ddfw.init_clause_weight | unsigned int  |  initial clause weight for DDFW local search | 8
 ddfw.reinit_base | unsigned int  |  increment basis for geometric backoff scheme of re-initialization of weights | 10000
-ddfw.restart_base | unsigned int  |  number of flips used a starting point for hessitant restart backoff | 100000
```

Comment:
these strings are extracted from z3 source. Spell check fixes are ported to master.
https://github.com/Z3Prover/z3/commit/3381fd2b52615189895a2704a474b9a5ba7637a4